docs/howto/getting-started/enable-openstack-cli.md

```diff
@@ -0,0 +1,164 @@
+# Enabling the OpenStack CLI tool
+
+The OpenStack CLI tool, also known as OpenStack Command Line Client, 
+conveniently provides access to various OpenStack APIs. Using the 
+OpenStack CLI tool, you can remotely create and manage the lifecycle of 
+objects relating, for example, to Compute, Networking, and Storage.
+
+Before installing `openstack` to your local laptop or workstation, you 
+first need to have an OpenStack user in your {{extra.brand}} account. 
+Next, you create and download a special RC file onto your computer, 
+modify it to reflect your OpenStack user's credentials, and source it. 
+Only then will you be able to use `openstack` --- provided it is 
+installed, of course.
+
+## Creating an OpenStack user
+
+From your favorite web browser, navigate to the [Cleura 
+Cloud](https://{{extra.gui_domain}}) page, and login into your 
+{{extra.brand}} account.
+
+Please make sure the left-hand side pane on the {{extra.gui}} is fully 
+visible, click the _Users_ category to expand it, and click on 
+_Openstack Users_.
+
+![Add a new OpenStack user](assets/ostack-cli/shot-01.png)
+
+Then, at the top right-hand side of the {{extra.gui}}, click once more 
+the _Add new Openstack user_ option. A new pane will slide into view, 
+titled _Create Openstack User_.
+
+![Create new OpenStack user](assets/ostack-cli/shot-02.png)
+
+Type in a username and a password for the new OpenStack user. To ensure 
+you typed the password correctly, you must re-type it below. This 
+password should be adequately strong, and thus a password manager may 
+come in handy.
+
+![Type-in username and password](assets/ostack-cli/shot-03.png)
```

Comment:
I think the alt text can lose the hyphen here — i.e. "Type in", not "Type-in". Or even just "Type" or "Set".